SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -746,6 +747,11 @@ void ASpatialFunctionalTest::PrepareTestAfterBeginPlay()
 	{
 		PrepareTest();
 	}
+
```

Comment:
Please can we add a comment here to say that this is for re-running tests?
So when we come back to this in the future we understand why this is needed.

Reply:
Added a comment